vmm/src/vm_config.rs

```diff
@@ -547,6 +547,12 @@ pub struct TpmConfig {
     pub socket: PathBuf,
 }
 
+#[derive(Clone, Debug, PartialEq, Eq, Deserialize, Serialize)]
+pub struct LandlockConfig {
```

Comment:
This also needs to be exposed via the http endpoint so that Kata and other orchestration layer can use it.

Reply:
Agreed. I have to figure how I can order this correctly. Meaning, I need to accept LandlockConfigs sent before `vm_create` and ignore configs sent after vm_create.

Reply:
On second thought, I don't see a lot of value in exposing this via http endpoint. Almost all the config specific rules are self-generated and handled. The only additional rules users have to pass are those for hot-adds. 

The Orchestration layer, can only pass landlock-rules before `vm_create`. It seems fair to let users start cloud-hypervisor with proper `--landlock-rules` and not worry about receiving additional rules via http endpoint.

Reply:
You can check with the Mariner team or the Kata community how this can be configured in their environment.

Reply:
> This also needs to be exposed via the http endpoint so that Kata and other orchestration layer can use it.

I see this like seccomp - this is vmm configuration not vm configuration and only the latter is exposed via the HTTP or D-Bus API.